### ISSUE TITLE
 Video recorded in macOS Chrome does not display in Firefox

### ISSUE THREAD

**Issue by debiig92** [collab-project/videojs-record#392]
## Description
When recording a video from Chrome and try to display it in firefox it shows the following message:
**Firefox Message**
**No video with supported format and MIME type found**
![image](https://user-images.githubusercontent.com/32922410/62235374-aad8d180-b381-11e9-8a33-7ab9708218e4.png)

**Firefox Warning:** 
![image](https://user-images.githubusercontent.com/32922410/62235605-2a66a080-b382-11e9-83d2-bd5b1ff461e6.png)
**Error Code: NS_ERROR_DOM_MEDIA_METADATA_ERR (0x806e0006)**

But when recording a video from firefox it displays well in chrome and firefox. 
My player init config is the following:

![image](https://user-images.githubusercontent.com/32922410/62236385-d8bf1580-b383-11e9-9073-a32df08c54e5.png)

I have no idea why this happens. Your answers would be very helpful. Thank you!

#### videojs
VIDEOJS: Using video.js 7.6.0 with videojs-record 3.8.0 and recordrtc 5.5.8
#### browsers
Firefox, Chrome
#### OSes
Ubuntu 18.04 (Production), Windows 10 (Development)

**Comment by gitsaket** [comment]
Hi @debiig92. I have the same issue but for mine, the videos recorded from the mac os was not playable over the FF. However, the windows and Linux machine recorded video over chrome was rendering with the FF. So, I have changed the videoMIMEType to 'video/webm;codecs=vp9'.

**Comment by debiig92** [comment]
I will try with video/webm;codecs=vp9 Thanks. Do you think It is a Firefox bug?

**Comment by ignl** [comment]
There is Firefox bug regarding mp4 incompatibility: https://bugzilla.mozilla.org/show_bug.cgi?id=1422891
Webm should work though.

**Comment by thijstriemstra** [comment]
> I will try with video/webm;codecs=vp9 Thanks. 

did this work @debiig92?

**Comment by thijstriemstra** [comment]
> vp8 should work though.

I recorded a video+audio clip in Ubuntu 20.04 using Chrome 89.0.4389 and was able to playback the clip using video.js in Firefox 85.0. Used videojs-record 4.2.0 and video.js 7.11.4.

>  I have the same issue but for mine, the videos recorded from the mac os was not playable over the FF. However, the windows and Linux machine recorded video over chrome was rendering with the FF.

I'll have to test macOS as well. Renaming the ticket.

**Comment by thijstriemstra** [comment]
The same issue applied to Safari as well and [since Safari Technology Preview 121](https://webkit.org/blog/11555/release-notes-for-safari-technology-preview-121/) that should be fixed:

- Fixed playback of WebM/Opus generated from Chrome MediaRecorder (r272822)

**Comment by thijstriemstra** [comment]
> I'll have to test macOS as well.

Tested with:

```
Google Chrome 89.0.4389.72 (Official Build) (x86_64)
macOS Version 11.2.2 (Build 20D80)
```

And saved the recorded webm file using `player.record().saveAs`. It saved the file as `foo12.webm;codecs=vp8,opus` which is a new issue, before it didn't append the `;codecs=vp8,opus` part. I opened a new ticket for this: #553

After renaming the file manually to `foo12.webm` I opened it in Firefox 86 and both audio and video properly displayed.

So considering this bug fixed.
